lib/cute_pets.rb

```diff
@@ -18,4 +19,17 @@ def post_pet()
       TweetGenerator.tweet(message, pet[:pic])
     end
   end
+
+  def post_pet_facebook()
+    pet = nil
+    if ENV.fetch('pet_datasource').downcase == 'petfinder'
```

Comment:
As this code is replicated in the twitter code, this looks like a good candidate to extract into a method for reuse.